src/components/MoneyRequestConfirmationList.js

```diff
@@ -466,7 +471,7 @@ function MoneyRequestConfirmationList(props) {
                     {props.isDistanceRequest ? (
                         <MenuItemWithTopDescription
                             shouldShowRightIcon={!props.isReadOnly && isTypeRequest}
-                            title={props.iouMerchant}
+                            title={doesRouteExist ? props.iouMerchant : translate('common.tbd')}
```

Comment:
Instead let's update where we get the distance request merchant to set only the amount to TBD.

https://github.com/Expensify/App/blob/1bdc5acc12f982b4f6e37bfadf9060ac3d2d3b3c/src/components/MoneyRequestConfirmationList.js#L298